bookkeeper-server/src/main/java/org/apache/bookkeeper/util/LocalBookKeeper.java

```diff
@@ -123,9 +123,7 @@ public static ZooKeeperServerShim runZookeeper(int maxCC, int zookeeperPort, Fil
 
         boolean b = waitForServerUp(InetAddress.getLoopbackAddress().getHostAddress() + ":" + zookeeperPort,
           CONNECTION_TIMEOUT);
-        if (LOG.isDebugEnabled()) {
-            LOG.debug("ZooKeeper server up: {}", b);
-        }
+        LOG.debug("ZooKeeper server up: {}", b);
```

Comment:
please keep `if (LOG.isDebugEnabled())`
`b` is boolean, log.debug takes Object as parameter sfo check for isDebugEnabled helps to avoid boxing.
It is not critical for the boolean per se (only two values/preallocated) but i'd keep it for the consistency with other places.

Reply:
OK